frontend/src/components/OrganizationForm/OrganizationForm.tsx

```diff
@@ -91,7 +91,9 @@ export const OrganizationForm: React.FC<{
       </DialogTitle>
       <DialogContent>
         <TextField
+          error={!values.name}
```

Comment:
I would do the following and ignore the red highlighting all together
```        <TextField
          margin="dense"
          helperText="Required Field"
          id="name"
          inputProps={{ maxLength: 250 }}
          name="name"
          label="Organization Name"
          required
          type="text"
          fullWidth
          value={values.name}
          onChange={onTextChange}
        />```

A user can't click Save without the organization name being completed then.